### PR TITLE
Ability to add additional files to product build.

### PR DESCRIPTION
**Pull request by aindlq**
This PR adds ability to add additional files to the product build.  As an example see updated `integrationTests/farm/` build.

### CODE REVIEW

File: libs/gretty/src/main/groovy/org/akhikhl/gretty/ProductConfigurer.groovy [hunk 1]
```diff
@@ -544,4 +550,13 @@ Version: ${project.version}"""
       file.text = fileText
     }
   }
+
+  protected void copyAdditionalFiles() {
+    Path outputPath = outputDir.toPath()
+	product.additionalFiles.each { source, target ->
```
Comment (boris-petrov) [inline libs/gretty/src/main/groovy/org/akhikhl/gretty/ProductConfigurer.groovy:556]: Indentation is wrong here.

File: libs/gretty/src/main/groovy/org/akhikhl/gretty/ProductConfigurer.groovy [hunk 2]
```diff
@@ -544,4 +550,13 @@ Version: ${project.version}"""
...
+      Path targetPath = outputPath.resolve(target)
+      Files.createDirectories(targetPath.getParent());
```
Comment (boris-petrov) [inline libs/gretty/src/main/groovy/org/akhikhl/gretty/ProductConfigurer.groovy:558]: Please remove the semi-colon.

File: integrationTests/farm/MyWebApp/build.gradle [hunk 3]
```diff
@@ -13,6 +13,11 @@ farm {
   webapp ':farm:MyWebService'
 }
 
+// adds additional file to the product build, for testing purpose
+product {
+  additionalFiles = ["./farm/README.md":"./README.md"]
```
Comment (boris-petrov) [inline integrationTests/farm/MyWebApp/build.gradle:18]: `additionalFiles = ['./farm/README.md': './README.md']`

**Comment by boris-petrov** [comment]
Thanks! I added a few code-style comments, please resolve them and then we can merge this.

**Comment by aindlq** [comment]
@boris-petrov thanks for review! I fixed the formatting and squashed the commits.

**Review by boris-petrov — APPROVED** [review]

**Review by f4lco — APPROVED** [review]

**Comment by boris-petrov** [comment]
@aindlq - thanks for the contribution!

**Pull request merged**